en_ewt-ud-dev.conllu

```diff
@@ -13034,7 +13034,7 @@
 8	looked	look	VERB	VBD	Mood=Ind|Number=Sing|Person=1|Tense=Past|VerbForm=Fin	2	conj	2:conj:and	_
 9	up	up	ADP	RP	_	8	compound:prt	8:compound:prt	_
 10	Rat	rat	NOUN	NN	Number=Sing	11	compound	11:compound	_
-11	ASCII's	ascii'	NOUN	NNS	Number=Plur	8	obj	8:obj	_
+11	ASCII's	ascii	NOUN	NNS	Number=Plur	8	obj	8:obj	_
```

Comment:
I think ASCII should be PROPN (and all-caps lemma)

Reply:
Sure, that makes sense